adafruit_pyportal.py

```diff
@@ -654,11 +654,11 @@ def image_converter_url(image_url, width, height, color_depth=16):
                                           width, height,
                                           color_depth, image_url)
 
-    def io_push(self, feed, data):
+    def io_push(self, feed_key, data):
         # pylint: disable=line-too-long
         """Push data to an adafruit.io feed
 
-        :param str feed: Name of feed to push data to.
+        :param str feed_key: Name of feed to push data to.
```

Comment:
Change to:
Name of feed key to push data to

Reply:
change made